### ISSUE TITLE
ParserError with DataRowVersion

### ISSUE THREAD

**Issue by wilsade** [icsharpcode/NRefactory#517]
Hi there.

I've got a parser error when parsing this code:

`    [TestMethod]
    public void WhenParsingThisCodeReturnNoError()
    {
      const string sampleCode = @"
using System.Data;
namespace Test
{
  public class TestClass
  {
    void TestMethod()
    {
      DataRow row = null;

      row[""AnyField""] = row.RowState == DataRowState.Deleted ? 
        row[""OtherField"", DataRowVersion.Original] : 
        row[""SomeField""];
    }
  }
}";
      var parser = new ICSharpCode.NRefactory.CSharp.CSharpParser();
      var syntaxTree = parser.Parse(sampleCode);
      Assert.IsTrue(syntaxTree.Errors.Count == 0, "Parser error");
    }`

If I remove the DataRowVersion, the parse is OK:

row[""AnyField""] = row.RowState == DataRowState.Deleted ? 
        row[""OtherField""] : 
        row[""SomeField""];

Thanks,
Wilsade

**Comment by fabiosfs** [comment]
It also happens in my app.
Is it going to be fixed?

**Comment by ibirite** [comment]
It seems that it happens only when you are using the ternary operator. But it's a bug, indeed.

**Comment by wilsade** [comment]
Hi.

Still no response?

Wilsade

**Comment by dgrunwald** [comment]
No, this is not going to be fixed. There is no one maintaining the NRefactory parser.
